### PR TITLE
Idr/form

### PR DESCRIPTION
**Pull request by Florida-Korir**
Updated UserRequestForm to TailwinCss

### CODE REVIEW

**Review by NewtonMutugi — APPROVED** [review]
LGTM

**Review by NewtonMutugi — APPROVED** [review]

File: tailwind.config.js
Comment (cbrianbet) [inline tailwind.config.js:1]: why did we edit this file?

Comment (cbrianbet) [reply tailwind.config.js:1]: @Florida-Korir

**Review by MaryKilewe — APPROVED** [review]

File: tailwind.config.js
Comment (MaryKilewe) [reply tailwind.config.js:1]: @Florida-Korir maybe you can check on this first. the rest seems fine

File: src/views/user/UserRequestForm.svelte [hunk 1]
```diff
@@ -0,0 +1,241 @@
+<script>
+    let files = [];
+    let deadline = '';
+    let textInput = '';
+  
+    const config = process.env.config;
+    const uploadUrl = config.UPLOAD_URL.toString();
```
Comment (NewtonMutugi) [inline src/views/user/UserRequestForm.svelte:7]: @Florida-Korir 
Remove the unnecessary `.toString()`. Let it just be:
```suggestion
    const uploadUrl = config.UPLOAD_URL;
```

**Review by NewtonMutugi — APPROVED** [review]

**Review by cbrianbet — APPROVED** [review]

**Pull request merged**